fhem/fhem.cfg

```
@@ -254,22 +254,30 @@ define FileLog_Balkonkraftwerk FileLog ./log/Balkonkraftwerk-%Y.log Balkonkraftw
 setuuid FileLog_Balkonkraftwerk 61b89822-f33f-ff84-f97c-7541cc463bf06c84
 attr FileLog_Balkonkraftwerk logtype text
 attr FileLog_Balkonkraftwerk room MQTT Geräte
-# Dummy Shelly
-define Balkon_In dummy
-setuuid Balkon_In 61b9b663-f33f-ff84-c962-d5828257071a7251
-attr Balkon_In readingList relay_0_power
-attr Balkon_In room Shelly Dummy
-attr Balkon_In setList on off
-attr Balkon_In useSetExtensions 1
-attr Balkon_In webCmd on:off
-define Balkon_Out dummy
-setuuid Balkon_Out 61b9b969-f33f-ff84-2e95-e28b6da93288d746
-attr Balkon_Out readingList relay_1_power
-attr Balkon_Out room Shelly Dummy
-attr Balkon_Out setList on off
-attr Balkon_Out useSetExtensions 1
-attr Balkon_Out userReadings UeberhitzungsSchutz
-attr Balkon_Out webCmd on:off
+#Solarstrom
+define Solarstrom dummy
+setuuid Solarstrom 61dafb08-f33f-26d3-073f-18ef2537a4984e95
+attr Solarstrom readingList relay_0_power
+attr Solarstrom room Shelly Dummy
+attr Solarstrom setList relay_0_power:textField
```

```suggestion
attr Solarstrom setList relay_0_power:slider,0,10,1000
```
Können auch einen Slider einbauen